pipelines/upgrade_pipeline.yml

```diff
@@ -31,12 +33,62 @@
     - disable_firewall
     - foreman_installer
 
-- hosts: "{{ forklift_server_name }}"
+- hosts:
+    - "{{ forklift_server_name }}"
+  become: yes
+  vars_files:
+    - vars/upgrade_base.yml
+    - vars/{{ pipeline_type }}_base.yml
+    - vars/{{ pipeline_type }}_base_release.yml
+    - vars/{{ pipeline_type }}_server.yml
+  roles:
+    - role: forklift_versions
+      scenario: "{{ pipeline_type }}"
+      scenario_version: "{{ pipeline_version }}"
+      upgrade: True
+    - role: forklift_versions
+      scenario: "{{ pipeline_type }}"
+      scenario_version: "{{ forklift_upgrade_version_start }}"
+    - foreman_installer
+
+- hosts:
+    - "{{ forklift_proxy_name }}"
+  become: yes
+  vars_files:
+    - vars/upgrade_base.yml
+    - vars/{{ pipeline_type }}_proxy.yml
+  vars:
+    foreman_proxy_content_server: "{{ forklift_server_name }}"
+    foreman_installer_disable_system_checks: true
+    foreman_installer_options_internal_use_only:
```

Comment:
Should this be moved to the `_proxy.yml` file as well to mirror the `_server.yml` file?

Reply:
No (well, maybe). `vars_files` has higher priority than `vars`, so if we define it in `proxy.yml`, it gets also passed on upgrades, which is strictly speaking not needed.

Reply:
I was thinking of having a different variable name on the initial install and using that variable in the pipeline on the install step. We can also do it later, but I'd like to add a proxy to the regular foreman as well, even though we'll have to kill the foreman-proxy-content role for it.

Reply:
I'd prefer to defer that to another PR